lib/cli/run.js

```diff
@@ -268,7 +269,7 @@ exports.builder = yargs =>
       }
 
       if (argv.compilers) {
-        throw createInvalidArgumentValueError(
+        throw createUnsupportedError(
           `--compilers is DEPRECATED and no longer supported.
           See ${ansi.cyan('https://git.io/vdcSr')} for migration information.`
```

Comment:
Don't think colorized formatting has any place in thrown error message.

Reply:
I think I agree.